README.md
# yeep
Yeep:: CMS SIN Data Base written in PHP

YEEP is a simply and minimalist web application written in php. This CMS is the perfect way to create your own website sin Data Base.



news.txt
a:10:{i:0;a:3:{s:5:"titre";s:18:"Bienvenue sur Yeep";s:6:"auteur";s:6:"Biwave";s:7:"contenu";s:239:"Hi, and welcome on Yeep. Yeep eis a website developped by Will C. Powered by PHP. iT CONSIST TO EDIT AND CREATE NEWS. 

Ainsi, ce moteur peut permettre de modifier des pages web ou des elements sur un site web. Bonne navigation sur Yeep!";}i:1;a:3:{s:5:"titre";s:15:"Welcome on Yeep";s:6:"auteur";s:6:"Biwave";s:7:"contenu";s:228:"Hello, welcome to YEEP. YEEP is a website developed by Biwave. This site is powered by a blog engine written in PHP. On this, you can write, edit part of text on this web page. To have more information, I am on Twitter: @enclize";}i:2;a:3:{s:5:"titre";s:0:"";s:6:"auteur";s:6:"Biwave";s:7:"contenu";s:63:"I\'m writing an Add-on on the website to create a writer acces.";}i:3;a:3:{s:5:"titre";s:0:"";s:6:"auteur";s:0:"";s:7:"contenu";s:27:"http://www.chrisj.com.au/
";}i:4;a:3:{s:5:"titre";s:0:"";s:6:"auteur";s:0:"";s:7:"contenu";s:175:"https://www.sites.google.com/site/premieresetterminales/classe-de-premiere/science-physique-de-premiere



     http://www.lachimie.fr/definitions/exercice-calcul-mole.php";}i:5;a:3:{s:5:"titre";s:0:"";s:6:"auteur";s:0:"";s:7:"contenu";s:449:"http://www.lyc-les-iscles.ac-aix-marseille.fr/spip/IMG/pdf/C01_FE_0910.pdf     http://www.ilephysique.net/forum-sujet-263414.html      http://www.lachimie.fr/definitions/exercice-calcul-mole.php  https://docs.google.com/viewer?a=v&pid=sites&srcid=ZGVmYXVsdGRvbWFpbnxwcmVtaWVyZXNldHRlcm1pbmFsZXN8Z3g6NDgyNDFjZDJiNDA0ODMwYw   https://docs.google.com/viewer?a=v&pid=sites&srcid=ZGVmYXVsdGRvbWFpbnxwcmVtaWVyZXNldHRlcm1pbmFsZXN8Z3g6NDgyNDFjZDJiNDA0ODMwYw";}i:6;a:3:{s:5:"titre";s:0:"";s:6:"auteur";s:0:"";s:7:"contenu";s:32:"http://www.csszengarden.com/220/";}i:8;a:3:{s:5:"titre";s:0:"";s:6:"auteur";s:6:"Biwave";s:7:"contenu";s:81:"http://www.wildbunny.co.uk/blog/2011/12/11/how-to-make-a-2d-platform-game-part-1/";}i:9;a:3:{s:5:"titre";s:0:"";s:6:"auteur";s:0:"";s:7:"contenu";s:43:"https://www.youtube.com/watch?v=5I7pukuy8sQ";}i:10;a:3:{s:5:"titre";s:0:"";s:6:"auteur";s:0:"";s:7:"contenu";s:75:"http://codes-sources.commentcamarche.net/source/52290-espace-membre-basique";}}

home.php
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="fr" lang="fr">
	<head>
		<title>Yeep::ACCES REDACTEUR</title>
		<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
	<link rel="stylesheet" href="styles.css" type="text/css" charset="utf-8" />
	</head>
	<body>
<center>
	<div id="header">
	<a href="../yeep/"><img
style="border: 0px solid ; width: 372px; height: 210px;" alt=""
src="../yeep/headr.png"></a> <a href="imgUpload/">Image Uploader</a>
</div>
<div id="content">	

   <?php
	if (isset($_POST['mot_de_passe']) AND $_POST['mot_de_passe'] ==  "720") // Si le mot de passe est bon
	{
	// On affiche les codes
	?>
		
			
		<p><em>Bienvenue sur Yeepr! Vous pouvez des maintenant modifier les acrticles du blog avec vos collaborateur sur ce board.</em></p>
		<p>_________________</p>
		<?php
$liste_news = unserialize(file_get_contents('news.txt'));
if(!empty($liste_news)) {
	foreach($liste_news as $id => $news) {
		$news = array_map('htmlspecialchars', $news);
		?>
		<h2><?php echo $news['titre'] ?></h2>
			<i>Ajout&eacute;e par <?php echo $news['auteur'] ?></i> <br />
			<p><?php echo $news['contenu'] ?></p>
			<i><a href="supprimer_news.php?id=<?php echo $id ?>" onclick="return confirm('Etes-vous s&ucirc;r de vouloir supprimer cette news ?');">Supprimer</a>
			&nbsp;
			<a href="editer_news.php?id=<?php echo $id ?>">Editer</a></i>
			<br /><br />
		<?php
	}
}
else {
	echo 'Il n\'y a aucune news pour le moment<br />';
}
echo '<a href="ajouter_news.php">Ajouter une news</a>';
?>

        <?php
	}
	else // Sinon, on affiche un message d'erreur
	{
		echo '<p>Deconnexion soudaine ou/et votre mot de passe est errone.</p>
		<p> Le site peut egalement etre en cours de maintenance. Veulliez reessayer plus tard. </p>
		<p> POWERED BY YEEP</p>';
	}
	?>
	
		<p>Initialement cree par William CLEMENT.</p>
		<p><strong>wiyam</strong></p>
		<p>(L) Yeep</p>
</div>
</center>		
	</body>
</html>

about.php
<!DOCTYPE html>
<html lang="en" class="no-js">

<head>

<!-- title and meta -->
<meta charset="utf-8" />
<meta name="viewport" content="width=device-width,initial-scale=1.0" />
<meta name="description" content="" />
<title>Yeep:: LE BLOG COMMUNAUTAIRE</title>
    
<!-- css -->
<link href='http://fonts.googleapis.com/css?family=Open+Sans:400italic,600italic,700italic,400,600,700' rel='stylesheet' type='text/css'>
<link href='http://fonts.googleapis.com/css?family=Bitter:400,700' rel='stylesheet' type='text/css'>
<link rel="stylesheet" href="css/font-awesome/font-awesome.css">
<link rel="stylesheet" href="css/base.css" />
<link rel="stylesheet" href="css/style.css" />

</head>



<body>

<nav class="menu slide-menu-left">
       <ul>
        <li><button class="close-menu"><h2>=</h2></button></li>
        <li><a href="#">Yeep</a></li>
		<li><a href="yeepr.php">Yeep acces redacteur</a></li>
		<li><a href="about.php">A propos</a></li>
		<li><a href="room.php">Tchat Room</a></li>
    </ul>
</nav><!-- /slide menu left -->

<nav class="menu slide-menu-right">
    <ul>
        <li><button class="close-menu">Close &darr;</button></li>
        <li><a href="http://native.chez.com/blog/">Native Open DATA</a></li>
        <li><a href="http://native.chez.com/tolki/">Tolki-THE GAME</a></li>
        <li><a href="http://native.chez.com/headr">HEADr</a></li>
        <li><a href="http://qappl.cla.fr/">QAPPL</a></li>
    </ul>
</nav><!-- /slide menu right -->

<nav class="menu slide-menu-top">
    <ul>
        <li><button class="close-menu">Close &darr;</button></li>
        <li><a href="http://native.chez.com/blog/">Native Open DATA</a></li>
        <li><a href="http://native.chez.com/tolki/">Tolki-THE GAME</a></li>
        <li><a href="http://native.chez.com/headr">HEADr</a></li>
        <li><a href="http://qappl.cla.fr/">QAPPL</a></li>
    </ul>
</nav><!-- /slide menu top -->

<nav class="menu slide-menu-bottom">
    <ul>
        <li><button class="close-menu">Close &darr;</button></li>
        <li><a href="http://native.chez.com/blog/">Native Open DATA</a></li>
        <li><a href="http://native.chez.com/tolki/">Tolki-THE GAME</a></li>
        <li><a href="http://native.chez.com/headr">HEADr</a></li>
        <li><a href="http://qappl.cla.fr/">QAPPL</a></li>
    </ul>
</nav><!-- /slide menu bottom -->

<nav class="menu push-menu-left">
    <ul>
        <li><button class="close-menu">Close &darr;</button></li>
        <li><a href="http://native.chez.com/blog/">Native Open DATA</a></li>
        <li><a href="http://native.chez.com/tolki/">Tolki-THE GAME</a></li>
        <li><a href="http://native.chez.com/headr">HEADr</a></li>
        <li><a href="http://qappl.cla.fr/">QAPPL</a></li>
    </ul>
</nav><!-- /push menu left -->

<nav class="menu push-menu-right">
    <ul>
        <li><button class="close-menu">Close &darr;</button></li>
        <li><a href="http://native.chez.com/blog/">Native Open DATA</a></li>
        <li><a href="http://native.chez.com/tolki/">Tolki-THE GAME</a></li>
        <li><a href="http://native.chez.com/headr">HEADr</a></li>
        <li><a href="http://qappl.cla.fr/">QAPPL</a></li>
    </ul>
</nav><!-- /push menu right -->

<nav class="menu push-menu-top">
    <ul>
        <li><button class="close-menu">Close &darr;</button></li>
        <li><a href="http://native.chez.com/blog/">Native Open DATA</a></li>
        <li><a href="http://native.chez.com/tolki/">Tolki-THE GAME</a></li>
        <li><a href="http://native.chez.com/headr">HEADr</a></li>
        <li><a href="http://qappl.cla.fr/">QAPPL</a></li>
    </ul>
</nav><!-- /push menu top -->

<nav class="menu push-menu-bottom">
    <ul>
        <li><button class="close-menu">Close &darr;</button></li>
        <li><a href="http://native.chez.com/blog/">Native Open DATA</a></li>
        <li><a href="http://native.chez.com/tolki/">Tolki-THE GAME</a></li>
        <li><a href="http://native.chez.com/headr">HEADr</a></li>
        <li><a href="http://qappl.cla.fr/">QAPPL</a></li>
    </ul>
</nav><!-- /push menu bottom -->

<div id="wrapper">

    <div class="info-bar">
        <div class="container">
            <a href="#" class="icon fa fa-home" data-title="#"></a>
            <a href="#" class="icon fa fa-arrow-left" data-title="#"></a>
            <!-- <a href="" class="icon fa fa-arrow-right" data-title="Next Demo"></a> -->
            <a href="#" class="icon fa fa-download" data-title="#"></a>
            <a href="#" class="icon fa fa-folder-open" data-title="#"></a>
        </div>
    </div><!-- /.info-bar -->
  
<center>  
    <header>
        <div class="branding">
            <div class="container clearfix">
                <div class="logo">
                    <a href="../yeep/">
                        <img src="img/logo.png" alt="Yeep" />
                    </a>
                </div>
                <div class="social">
                    
                    <a href="https://twitter.com/wiyam_" target="_blank" class="twitter">twitter</a>
                    
                </div>
            </div>
        </div><!-- /.branding -->
        <div class="site-title">
            <div class="container">
                <h1>Yeep </h1>
            </div>
        </div>
    </header>
</center>

    <div id="main">
        <div class="container">
            
            <div class="buttons">
                <button class="nav-toggler toggle-slide-left"><h2>= DISCOVER</h2></button>
				<div id="id_div_1" style="display:none;">
                <button class="nav-toggler toggle-slide-right"></button>
                <button class="nav-toggler toggle-slide-top"></button>
                <button class="nav-toggler toggle-slide-bottom"></button><br>
                <button class="nav-toggler toggle-push-left"></button>
                <button class="nav-toggler toggle-push-right"></button>
                <button class="nav-toggler toggle-push-top"></button>
                <button class="nav-toggler toggle-push-bottom"></button>
				</div>
            </div><!-- /buttons -->

            <section class="content">
                <h1>BLOG</h1>
<p>Yeep est un blog propulse par Yeepr, un CMS cree par Will "wiyam" C.</p>
<p>Ce blog est a la fois experimentale, mais les post postees sur ce blog sont egalement crees pour proposer un contenu intuitif portant sur differentes categories: internet, informatique, qui sont mes passion.</p>
<p>Vous pouvez me suivre sur Twitter: @wiyam_</p>
<p>Bonne navigation sur Yeep!</p>
<p>PS: je sais que les accents ne s'affichent pas, je suis sur le point de resoudre ce probleme.</p>
            </section>

        </div>
    </div><!-- #main -->


    <footer>
        <div class="container">
            <div class="clearfix">
                <aside>
                    <ul>
                        <li><a href="http://www.twitter.com/wiyam_/">TweetME</a></li>
                        
                    </ul>
                </aside>
                <aside>
                    <ul>
                        <li><a href="http://native.chez.com/about.html"></a></li>
                        
                    </ul>
                </aside>
            </div>
            <div class="copyright">
                <span>
                     Produced by WAC with LOVE<br>
                    <a href="#" target="_blank">#</a>
                </span>
            </div>
        </div>
    </footer><!-- /footer -->

</div><!-- /#wrapper -->

<!-- js -->
<script src="js/classie.js"></script>
<script src="js/nav.js"></script>

<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-34160351-1']);
  _gaq.push(['_trackPageview']);
  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script>

</body>
</html>

yeepr.php
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="fr" lang="fr">
	<head>
		<title>YEEPr</title>
		<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
	<link rel="stylesheet" href="styles.css" type="text/css" charset="utf-8" />
	</head>
	<body>
<center>
	<div id="toolbar">
	<a href="../yeep/"><img
style="border: 0px solid ; width: 372px; height: 210px;" alt=""
src="http://native.chez.com/yeep/headr.png"></a>
</div>
</center>
<center>
<div id="content">		
		<p>.</p>
		<p>.</p>
		<p>.</p>
		<p>.</p>
		<p>.</p>
		<p>.</p>
		<p>Board partcipatif</p>
		<p>Partager votre savoir avec des collaborateurs</p>
		<p>Un espace de travail ou de detente</p>
		<p>Un espace securise et unique reserve pour un groupe de pair, ou a un cercle strict de connaissance</p>
		<p>Vous seul connaissez le mot de passe de votre acces redacteur</p>
		<p><em>YEEPr est un board participatif permettant a des groupes d'utilisateurs de partager des informations et des donnees.</em></p>
		<p><em>Veuillez entrer le mot de passe pour acceder au systeme</em></p>
		<form action="home.php" method="post">
			<p>
			<input type="password" name="mot_de_passe" />
			<input type="submit" value="Valider" />
			</p>
		</form>
		<p>Initialement cree par William CLEMENT.</p>
		<p><strong>@wiyam_</strong></p>
		<p>(L) Yeep</p>
</div>
</center>		
	</body>
</html>

imgUpload/index.php
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><head><title>Nanorift</title>
	<style>
input[type="file"] {
  background-color:white; 
  font:bold 14px Arial, Helvetica, sans-serif; 
  color:#3287C8; 
  border:0; 
  cursor:pointer; 
}
.myButton {
	-moz-box-shadow: 0px 1px 0px 0px #f0f7fa;
	-webkit-box-shadow: 0px 1px 0px 0px #f0f7fa;
	box-shadow: 0px 1px 0px 0px #f0f7fa;
	background:-webkit-gradient(linear, left top, left bottom, color-stop(0.05, #33bdef), color-stop(1, #019ad2));
	background:-moz-linear-gradient(top, #33bdef 5%, #019ad2 100%);
	background:-webkit-linear-gradient(top, #33bdef 5%, #019ad2 100%);
	background:-o-linear-gradient(top, #33bdef 5%, #019ad2 100%);
	background:-ms-linear-gradient(top, #33bdef 5%, #019ad2 100%);
	background:linear-gradient(to bottom, #33bdef 5%, #019ad2 100%);
	filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#33bdef', endColorstr='#019ad2',GradientType=0);
	background-color:#33bdef;
	-moz-border-radius:6px;
	-webkit-border-radius:6px;
	border-radius:6px;
	border:1px solid #057fd0;
	display:inline-block;
	cursor:pointer;
	color:#ffffff;
	font-family:Trebuchet MS;
	font-size:14px;
	font-weight:bold;
	padding:10px 24px;
	text-decoration:none;
	text-shadow:0px -1px 0px #5b6178;
}
.myButton:hover {
	background:-webkit-gradient(linear, left top, left bottom, color-stop(0.05, #019ad2), color-stop(1, #33bdef));
	background:-moz-linear-gradient(top, #019ad2 5%, #33bdef 100%);
	background:-webkit-linear-gradient(top, #019ad2 5%, #33bdef 100%);
	background:-o-linear-gradient(top, #019ad2 5%, #33bdef 100%);
	background:-ms-linear-gradient(top, #019ad2 5%, #33bdef 100%);
	background:linear-gradient(to bottom, #019ad2 5%, #33bdef 100%);
	filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#019ad2', endColorstr='#33bdef',GradientType=0);
	background-color:#019ad2;
}
.myButton:active {
	position:relative;
	top:1px;
}

	</style></head><body> <div style="text-align: center;"><br><br><br><br><a href="index.php"><img style="border: 0px solid ; width: 250px; height: 244px;" alt="" src="logo.png"></a><br><br>
<a href="../">Yeep::site internet</a><br>
<a href="../yeepr.php">yeepr</a><br>

<form action="upload.php" method="post" enctype="multipart/form-data"> <input name="uploadedFile" id="uploadedFile" type="file"> <input class="myButton" value="Telecharger" name="submit" type="submit"> </form>

</div>


</body></html>


imgUpload/upload.php
<!DOCTYPE html>
<html>
    <head><TITLE>Nanorift
    </TITLE>
	</head>
    <body> 
	<div style="text-align: center;"><br><br><br><br><br><br><a href="index.php"><img style="width: 20%; height: auto;" alt="" src="logo.png"></a><br><br>
        <?PHP
        
        // 
        $MAX_FILE_SIZE = 2000000;
        $MAX_FILE_SIZE_STR = '2MB';
        $ACCEPTED_FILE_TYPES = array(IMAGETYPE_JPEG, IMAGETYPE_PNG);
        $ACCEPTED_FILE_TYPES_STR = 'JPG, PNG';
        $IMG_FRAME_WIDTH = 400;
        $IMG_FRAME_HEIGHT = 300;
        $IMG_FRAME_ASPECT = $IMG_FRAME_WIDTH / $IMG_FRAME_HEIGHT;
        
        // 
        $tempFileLocation = $_FILES['uploadedFile']['tmp_name'];
        @$imageInfo = getimagesize($tempFileLocation);
        
        // 
        $fileSize = $_FILES['uploadedFile']['size'];
        if ($fileSize > $MAX_FILE_SIZE) {
            echo "<p>Fichier trop gros</p>";
            echo "<p>Maximum: {$MAX_FILE_SIZE_STR}.</p>";
            exit();
        }

        // 
        $imageFileType = $imageInfo[2];
        if (!in_array($imageFileType, $ACCEPTED_FILE_TYPES)) {
            echo "<p>Fichiers invalides</p>";
            echo "<p>les fichiers acceptes sont: {$ACCEPTED_FILE_TYPES_STR}</p>";
            exit();
        }

        // 
        $targetFileName = $_FILES['uploadedFile']['name'];
        $targetFileLocation = "uploads/{$targetFileName}";
        $moveSuccess = move_uploaded_file($tempFileLocation, $targetFileLocation);
        if (!$moveSuccess) {
            echo "<p>ERREUR INTERNE</p>";
            exit();
        }

        // 
        // 
        $imageWidth = $imageInfo[0];
        $imageHeight = $imageInfo[1];
        $imageAspectRatio = $imageWidth / $imageHeight;

        if ($imageAspectRatio >= $IMG_FRAME_ASPECT) {
            $imageSizeAttr = "width='$IMG_FRAME_WIDTH'";
        } elseif ($imageAspectRatio < $IMG_FRAME_ASPECT) {
            $imageSizeAttr = "height='$IMG_FRAME_HEIGHT'";
        }

        $imageFrame = <<<HTML
                <div  style='width:{$IMG_FRAME_WIDTH}px; max-width:{$IMG_FRAME_WIDTH}px; height:{$IMG_FRAME_HEIGHT}px; border-style:transparent; position:center;'>
                <img src='$targetFileLocation' alt='Uploaded Image' {$imageSizeAttr}>
                </div>
HTML;
        echo $imageFrame;
		//
  $files = glob("uploads/*.*");

  for ($i=1; $i<count($files); $i++)

{

$image = $files[$i];
$supported_file = array(
    'gif',
    'jpg',
    'jpeg',
    'png'
);

$ext = strtolower(pathinfo($image, PATHINFO_EXTENSION));
if (in_array($ext, $supported_file)) {
    print $image ."<br />";
    echo '<img style="height:100px;weidth:auto;" src="'.$image .'" alt="Random image" />'."<br /><br />";
} else {
    continue;
 }

}
        ?>
		</div>
    </body>
</html>


room.php
<!DOCTYPE html>
<html lang="en" class="no-js">

<head>

<!-- title and meta -->
<meta charset="utf-8" />
<meta name="viewport" content="width=device-width,initial-scale=1.0" />
<meta name="description" content="" />
<title>Yeep:: Tchat Room</title>
    
<!-- css -->
<link href='http://fonts.googleapis.com/css?family=Open+Sans:400italic,600italic,700italic,400,600,700' rel='stylesheet' type='text/css'>
<link href='http://fonts.googleapis.com/css?family=Bitter:400,700' rel='stylesheet' type='text/css'>
<link rel="stylesheet" href="css/font-awesome/font-awesome.css">
<link rel="stylesheet" href="css/base.css" />
<link rel="stylesheet" href="css/style.css" />

</head>



<body>

<nav class="menu slide-menu-left">
    <ul>
        <li><button class="close-menu"><h2>=</h2></button></li>
        <li><a href="#">Yeep</a></li>
		<li><a href="yeepr.php">Yeep acces redacteur</a></li>
		<li><a href="about.php">A propos</a></li>
		<li><a href="room.php">Tchat Room</a></li>
    </ul>
</nav><!-- /slide menu left -->

<nav class="menu slide-menu-right">
    <ul>
        <li><button class="close-menu">Close &darr;</button></li>
        <li><a href="http://native.chez.com/blog/">Native Open DATA</a></li>
        <li><a href="http://native.chez.com/tolki/">Tolki-THE GAME</a></li>
        <li><a href="http://native.chez.com/headr">HEADr</a></li>
        <li><a href="http://qappl.cla.fr/">QAPPL</a></li>
    </ul>
</nav><!-- /slide menu right -->

<nav class="menu slide-menu-top">
    <ul>
        <li><button class="close-menu">Close &darr;</button></li>
        <li><a href="http://native.chez.com/blog/">Native Open DATA</a></li>
        <li><a href="http://native.chez.com/tolki/">Tolki-THE GAME</a></li>
        <li><a href="http://native.chez.com/headr">HEADr</a></li>
        <li><a href="http://qappl.cla.fr/">QAPPL</a></li>
    </ul>
</nav><!-- /slide menu top -->

<nav class="menu slide-menu-bottom">
    <ul>
        <li><button class="close-menu">Close &darr;</button></li>
        <li><a href="http://native.chez.com/blog/">Native Open DATA</a></li>
        <li><a href="http://native.chez.com/tolki/">Tolki-THE GAME</a></li>
        <li><a href="http://native.chez.com/headr">HEADr</a></li>
        <li><a href="http://qappl.cla.fr/">QAPPL</a></li>
    </ul>
</nav><!-- /slide menu bottom -->

<nav class="menu push-menu-left">
    <ul>
        <li><button class="close-menu">Close &darr;</button></li>
        <li><a href="http://native.chez.com/blog/">Native Open DATA</a></li>
        <li><a href="http://native.chez.com/tolki/">Tolki-THE GAME</a></li>
        <li><a href="http://native.chez.com/headr">HEADr</a></li>
        <li><a href="http://qappl.cla.fr/">QAPPL</a></li>
    </ul>
</nav><!-- /push menu left -->

<nav class="menu push-menu-right">
    <ul>
        <li><button class="close-menu">Close &darr;</button></li>
        <li><a href="http://native.chez.com/blog/">Native Open DATA</a></li>
        <li><a href="http://native.chez.com/tolki/">Tolki-THE GAME</a></li>
        <li><a href="http://native.chez.com/headr">HEADr</a></li>
        <li><a href="http://qappl.cla.fr/">QAPPL</a></li>
    </ul>
</nav><!-- /push menu right -->

<nav class="menu push-menu-top">
    <ul>
        <li><button class="close-menu">Close &darr;</button></li>
        <li><a href="http://native.chez.com/blog/">Native Open DATA</a></li>
        <li><a href="http://native.chez.com/tolki/">Tolki-THE GAME</a></li>
        <li><a href="http://native.chez.com/headr">HEADr</a></li>
        <li><a href="http://qappl.cla.fr/">QAPPL</a></li>
    </ul>
</nav><!-- /push menu top -->

<nav class="menu push-menu-bottom">
    <ul>
        <li><button class="close-menu">Close &darr;</button></li>
        <li><a href="http://native.chez.com/blog/">Native Open DATA</a></li>
        <li><a href="http://native.chez.com/tolki/">Tolki-THE GAME</a></li>
        <li><a href="http://native.chez.com/headr">HEADr</a></li>
        <li><a href="http://qappl.cla.fr/">QAPPL</a></li>
    </ul>
</nav><!-- /push menu bottom -->

<div id="wrapper">

    <div class="info-bar">
        <div class="container">
            <a href="#" class="icon fa fa-home" data-title="#"></a>
            <a href="#" class="icon fa fa-arrow-left" data-title="#"></a>
            <!-- <a href="" class="icon fa fa-arrow-right" data-title="Next Demo"></a> -->
            <a href="#" class="icon fa fa-download" data-title="#"></a>
            <a href="#" class="icon fa fa-folder-open" data-title="#"></a>
        </div>
    </div><!-- /.info-bar -->
  
<center>  
    <header>
        <div class="branding">
            <div class="container clearfix">
                <div class="logo">
                    <a href="../yeep/">
                        <img src="img/logo.png" alt="Yeep" />
                    </a>
                </div>
                <div class="social">
                    
                    <a href="https://twitter.com/wiyam_" target="_blank" class="twitter">twitter</a>
                    
                </div>
            </div>
        </div><!-- /.branding -->
        <div class="site-title">
            <div class="container">
                <h1>Yeep </h1>
            </div>
        </div>
    </header>
</center>

    <div id="main">
        <div class="container">
            
            <div class="buttons">
                <button class="nav-toggler toggle-slide-left"><h2>= DISCOVER</h2></button>
				<div id="id_div_1" style="display:none;">
                <button class="nav-toggler toggle-slide-right"></button>
                <button class="nav-toggler toggle-slide-top"></button>
                <button class="nav-toggler toggle-slide-bottom"></button><br>
                <button class="nav-toggler toggle-push-left"></button>
                <button class="nav-toggler toggle-push-right"></button>
                <button class="nav-toggler toggle-push-top"></button>
                <button class="nav-toggler toggle-push-bottom"></button>
				</div>
            </div><!-- /buttons -->

            <section class="content">
                <h1>BLOG</h1>
<p>Yeep TchatRoom vous propose de discuter avec d'autres lecteurs du blog.</p>

<div id="tlkio" data-channel="yeep" style="width:100%;height:700px;"></div><script async src="http://tlk.io/embed.js" type="text/javascript"></script>

            </section>

        </div>
    </div><!-- #main -->


    <footer>
        <div class="container">
            <div class="clearfix">
                <aside>
                    <ul>
                        <li><a href="http://www.twitter.com/wiyam_">TweetME</a></li>
                        
                    </ul>
                </aside>
                <aside>
                    <ul>
                        <li><a href="http://native.chez.com/about.html"></a></li>
                        
                    </ul>
                </aside>
            </div>
            <div class="copyright">
                <span>
                     Produced by WAC with LOVE<br>
                    <a href="#" target="_blank">#</a>
                </span>
            </div>
        </div>
    </footer><!-- /footer -->

</div><!-- /#wrapper -->

<!-- js -->
<script src="js/classie.js"></script>
<script src="js/nav.js"></script>

<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-34160351-1']);
  _gaq.push(['_trackPageview']);
  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script>

</body>
</html>